pkgs/development/ruby-modules/bundler-app/default.nix

```diff
@@ -15,7 +15,7 @@
   # gemdir is the location of the Gemfile{,.lock} and gemset.nix; usually ./.
 , gemdir
   # Exes is the list of executables provided by the gems in the Gemfile
-, exes ? []
+, exes ? [ pname ]
```

Comment:
Not sure how I feel about this… There's not necessarily any reason for the executable name to be the same as the package name. IMHO it should be explicit.

Reply:
As long as the build fails if it's not the case it should be alright. But yeah, one more heuristic.

Reply:
No problem, I thought it was a safe default, but it's hardly necessary.